tests/destructor/tbintree2.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   cmd: '''nim c -d:nimAllocStats --newruntime $file'''
+  disabled: "true"
```

Comment:
This example is valuable to keep active.

Reply:
It doesn't work with `--gc:arc`.

Reply:
The example doesn't work for `--gc:arc/orc`.

Reply:
At least keep it enabled then?

Reply:
But `newruntime` is a nope now.

Reply:
Don't make a nope, the switch should be undocumented but for tests like this it's quite valuable -- we don't test "move only" types all that well otherwise.

Reply:
I see

Reply:
ya, the deprecation as done in newer version of this PR seems better than turning it into noop; this buys more time.